apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -13,60 +13,31 @@
 // limitations under the License.
 
 using Google.Api.Gax;
-using Newtonsoft.Json;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
-using System.Reflection;
-using System.Text;
 using System.Web;
 
+using LabelsCommon = Google.Cloud.Diagnostics.Common.Labels;
+
 namespace Google.Cloud.Diagnostics.AspNet
 {
     /// <summary>
     /// A helper class to handle span labels.
     /// </summary>
     internal static class Labels
-    {
-        /// <summary>The version of the trace agent.</summary>
-        internal static readonly string AgentVersion = typeof(Labels)
-            .GetTypeInfo()
-            .Assembly
-            .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
-            .InformationalVersion;
-
+    { 
         /// <summary>
         /// Gets a map with the label for the agent which contains the agent's name and version.
         /// </summary>
         internal static Dictionary<string, string> AgentLabel { get; } =
-            new Dictionary<string, string> { { Agent, $"{AgentName} {AgentVersion}" } };
-
-        /// <summary>The name of the trace agent.</summary>
-        internal const string AgentName = "google-cloud-csharp-trace";
-
-        ///<summary>The label to denote the size of a request.</summary> 
-        internal const string HttpRequestSize = "trace.cloud.google.com/http/request/size";
-
-        ///<summary>The label to denote the host.</summary> 
-        internal const string HttpHost = "trace.cloud.google.com/http/host";
-
-        ///<summary>The label to denote the request method.</summary> 
-        internal const string HttpMethod = "trace.cloud.google.com/http/method";
-
-        ///<summary>The label to denote the response status code.</summary> 
-        internal const string HttpStatusCode = "trace.cloud.google.com/http/status_code";
-
-        ///<summary>The label to denote a stack trace.</summary> 
-        internal const string StackTrace = "trace.cloud.google.com/stacktrace";
-
-        ///<summary>The label to denote an agent.</summary> 
-        internal const string Agent = "/agent";
-
+            LabelsCommon.GetAgentLabel(typeof(Labels));
+        
         /// <summary>
         /// Gets a map of labels for a span from an <see cref="HttpRequest"/>, such as request size, method, ect.
         /// </summary>
         public static Dictionary<string, string> FromHttpRequest(HttpRequest request)
         {
+            GaxPreconditions.CheckNotNull(request, nameof(request));
```

Comment:
If you make this expression the argument to the `HttpRequestWrapper` constructor, this can be an expression-bodied method. Up to you though. (same comment below)